### ISSUE TITLE
How it works?

### ISSUE THREAD

**Issue by Helias** [zZeck/raymanarenadecomp#1]
how can I use?
it decompiles rayman arena at 100% and make it work natively on your machine?

is there a better documentation to use it? maybe on Linux?

**Comment by zZeck** [comment]
The project has barely started. I have only decompiled one routine. It works by using Frida to hook the original exe and diverting execution to the replacement. Until a lot more of the game is decompiled, running on linux directly is not possible (although it may work with Wine). I also began work on automated tests for the decompiled routine.

I'm currently working on game file manipulation tools. Raymap can load levels, but not edit them. I decided getting a firm understanding of the files and then their in memory representation would be the most useful thing I could do first. Nothing is at the point it is [useful yet](https://github.com/zZeck/raymanarenatool).

There is a [community with discord](https://discord.gg/raymanarenacommunity) playing the game online using a [fix to the LAN mode](https://github.com/zZeck/RaymanArenaPatch) I made and later enhanced with discord presence support.
